packages/orchestration/src/orchestration-types.ts

```diff
@@ -63,6 +63,21 @@ export interface OrchestrationModuleConfig {
   llm: LlmModuleConfig;
   /**
    * Filtering module configuration.
+   * @example
```

Comment:
[req] I will keep the example not so long, showing how it's used in input is enough since it's basically the same in output. I would instead mention in the description that the configuration is for input and output and filters can be applied on both